estate/models/estate_property.py

```diff
@@ -0,0 +1,93 @@
+from odoo import _, api, fields, models, tools
+from odoo.exceptions import UserError
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "This model stores estate properties information and the current state of the sales process"
+    _order = "id desc"
+    _sql_constraints = [
+        ('expected_price_check', 'CHECK (expected_price > 0)', 'the expected price must be strictly positive'),
+        ('selling_price_check', 'CHECK (selling_price >= 0)', 'the selling price must be positive')
+    ]
+
+    buyer_id = fields.Many2one("res.partner", string="Buyer", copy=False)
+    property_type_id = fields.Many2one("estate.property.type", string="Property Type")
+    salesperson_id = fields.Many2one("res.users", string="Seller", default=lambda self: self.env.user)
+    offer_ids = fields.One2many("estate.property.offer", "property_id")
+    property_tag_ids = fields.Many2many("estate.property.tag")
+
+    name = fields.Char(string="Title", required=True)
+    description = fields.Text(string="Description")
```

Comment:
Why this division between relational fields and the others ?

Reply:
I've divided them because I thought it would be clearer. They are important because they are linked to other models and therefore I wanted to emphasise them.